src/cosalib/kubevirt.py

```diff
@@ -42,41 +41,22 @@ def write_oci(self, image_name):
 
 
 def kubevirt_run_ore(build, args):
-    if not args.repository:
-        raise Exception("--repository must not be empty")
-
-    name = f"{build.build_name}"
-    if args.name is not None:
-        name = args.name
-    tags = [f"{build.build_id}-{build.basearch}"]
-    if args.tag is not None:
-        tags.extend(args.tag)
-    full_name = os.path.join(args.repository, name)
-
-    digest = runcmd(["skopeo", "inspect", f"oci-archive:{build.image_path}", "-f", "{{.Digest}}"],
-                    stdout=subprocess.PIPE).stdout.decode("utf-8").strip()
-    for tag in tags:
-        log.info(f"pushing {full_name}:{tag} with digest {digest}")
-        runcmd(["skopeo", "copy", f"oci-archive:{build.image_path}", f"docker://{full_name}:{tag}"])
-
-    build.meta['kubevirt'] = {
-        'image': f"{full_name}@{digest}",
-    }
-    build.meta_write()
+    """
+    This function is a NOOP. We'll push the ociarchive files using
```

Comment:
Minor: The noop comment is a bit weird here since we're raising an exception. I know what you mean, but maybe a more accurate first sentence could be "This function is not necessary for kubevirt."? And similarly below.

Reply:
done